<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(231, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Thông tin khách hàng&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã Thuê</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên Khách Hàng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Số Điện Thoại</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Địa Chỉ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaThue_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Phone_text">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TenKhach_text">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DiaChi_text">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Them_KH">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 255, 127)</string>
    </property>
    <property name="text">
     <string>Thêm</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../test/them.webp</normaloff>../test/them.webp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Sua_KH">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 170, 255)</string>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../test/sua.webp</normaloff>../test/sua.webp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Xoa_KH">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../test/xoa.webp</normaloff>../test/xoa.webp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 127)</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../test/home-64.webp</normaloff>../test/home-64.webp</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table_KH">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>661</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
